test/index.js

```diff
@@ -44,13 +48,14 @@ function triggerClick(el, modified){
             window, null,
             0, 0, 0, 0, /* coordinates */
             !!modified, false, false, false, /* modifier keys */
-            0 /*left*/,
+            button,
             null
         );
         el.dispatchEvent(ev);
     } else if (document.createEventObject) {
         ev = document.createEventObject();
         ev.ctrlKey = !!modified;
+        ev.button = button;
```

Comment:
That's where I was looking too. I've got access to IE through saucelabs so I'll do some research on `createEventObject` and see if I can get it to pass.